test/unit/src/index-paginator-test.mjs

```diff
@@ -432,6 +439,356 @@ test('Async iterator - plugin', async t => {
   reset()
 })
 
+test('Default paginator - raw client', async t => {
+  let aws = await client(config)
+  let headers = copy(jsonHeaders)
+  let response
+  let expectedResponsePayload
+  let requests
+
+  const simpleResponses = [
+    {
+      responseBody: simpleResponseBodies[0],
+      responseHeaders: jsonHeaders,
+    },
+    {
+      responseBody: simpleResponseBodies[1],
+      responseHeaders: jsonHeaders,
+    },
+    {
+      responseBody: simpleResponseBodies[2],
+      responseHeaders: jsonHeaders,
+    },
+  ]
+
+  const nestedResponses = [
+    {
+      responseBody: nestedResponseBodies[0],
+      responseHeaders: jsonHeaders,
+    },
+    {
+      responseBody: nestedResponseBodies[1],
+      responseHeaders: jsonHeaders,
+    },
+    {
+      responseBody: nestedResponseBodies[2],
+      responseHeaders: jsonHeaders,
+    },
+  ]
+
+  t.test('Query cursor', async t => {
+    t.plan(4)
+    const responseGenerator = makeResponses(simpleResponses)
+    let expectedUrl
+    expectedResponsePayload = { Accumulator: [ 'a1', 'a2', 'a3' ] }
+
+    server.use({ accumulateRequests: true })
+    server.use({ responseGenerator })
```

Comment:
Any particular reason not to do `server.use({ accumulateRequests: true, responseGenerator })`?